scripts/pscis_tidy.R

```diff
@@ -41,7 +41,10 @@ utm <- 10
 #   ) %>%
 #   sf::st_write('data/inputs_extracted/mergin_backups/form_pscis_v70.gpkg', append=FALSE)
 
-# read in cleaned amalgamated pscis form
+
+#---------------------pscis clean and QA only--------------------------
+
+# read in amalgamated pscis form
 form_pscis <- sf::st_read(dsn= paste0('../../gis/', dir_project, '/data_field/2023/form_pscis_2023.gpkg')) %>%
   st_transform(crs = 26900 + utm) %>%
   poisspatial::ps_sfc_to_coords(X = 'easting', Y = 'northing') %>%
```

Comment:
i may just be missing it but I don't see `x` and `y` columns that are in a crs that works for all of BC so things don't break in the future when we have sites in more than one utm zone.

![image](https://github.com/NewGraphEnvironment/fish_passage_peace_2023_reporting/assets/46538103/b12be3cc-0036-4c16-bf0a-481dc121594d)